package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "alpheios-alignment-editor",
-  "version": "1.0.2",
+  "version": "1.2.2",
```

Comment:
When I run `npm install` on this branch, my `version` is set to `"1.3.0"`. Do you know why this would be happening? Should I be running a different command?

Reply:
it is the current version - so it is correct (but I would update it, because I forgot to do this).
I could suggest the following:
- before commit "resolve merge conflict" there was a version 1.1.2
- then it was updated to 1.3.0
- you didn't refresh the changes - that's why there was this difference